eng/cake/dotnet.cake

```diff
@@ -205,6 +208,44 @@ Task("dotnet-pack")
         Zip(assetsDir, $"{assetsDir}.zip");
     });
 
+Task("dotnet-pack-library-packs")
+    .Does(() =>
+    {
+        var destDir = $"./artifacts/additional-library-packs";
+
+        void Download(string id, string version, params string[] sources)
+        {
+            var tempDir = $"./artifacts/additional-library-packs-temp";
+            EnsureDirectoryExists(destDir);
+
+            version = XmlPeek("./eng/Versions.props", "/Project/PropertyGroup/" + version);
+
+            NuGetInstall(id, new NuGetInstallSettings
+            {
+                Version = version,
+                ExcludeVersion  = true,
+                OutputDirectory = tempDir,
+                Source = sources,
+            });
+
+            CopyFiles($"{tempDir}/**/" + id + ".nupkg", destDir, false);
+            CleanDirectories(tempDir);
+        }
+
+        Download("Microsoft.Maui.Graphics", "MicrosoftMauiGraphicsVersion", "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json");
+        Download("Microsoft.Maui.Graphics.Win2D.WinUI.Desktop", "MicrosoftMauiGraphicsVersion", "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json", "https://api.nuget.org/v3/index.json");
+        Download("Microsoft.WindowsAppSDK", "MicrosoftWindowsAppSDKPackageVersion", "https://api.nuget.org/v3/index.json");
+        Download("Microsoft.Windows.SDK.BuildTools", "MicrosoftWindowsSDKBuildToolsPackageVersion", "https://api.nuget.org/v3/index.json");
+
+        Zip(destDir, $"{destDir}.zip");
```

Comment:
Do we need to zip/unzip these? Should we maybe just upload them as a separate `library-packs` azure artifact to save a couple steps?

Reply:
Once we add a new artifact for this, we should add another download step to the top of the sdk-insertion.yml job:

```yaml
  - task: DownloadPipelineArtifact@2
    inputs:
      artifactName: library-packs
      downloadPath: $(Build.StagingDirectory)\nuget
      patterns: |
        **/Microsoft.Maui.Graphics.*.nupkg
```